app/main.py

```diff
@@ -1,3 +1,56 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: int | float) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int | float) -> "Distance":
```

Comment:
```suggestion
    def __mul__(self, other: int | float | Distance) -> Distance:
```
please fix everywhere 
please read https://github.com/mate-academy/py-distance-class/blob/master/checklist.md

Reply:
how to do that? I tried to fix this but the last I have a problem : Unresolved reference 'Distance'

Reply:
Thank you for help me with this part of code, I forgot to do this import(